projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,141 +2,96 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
-## 14.0-beta - 2024-11-04
-### Improved compatibility
-- Embed Blocks: Append 'Embed' to the titles of Facebook and Instagram embed blocks. [#39970]
-- Image Block: Make the external media picker button consistent with the other buttons in the image block. [#39985]
-
-### Bug fixes
-- Google Fonts: Fix module not loading fully in certain scenarios. [#39977]
-- Social: Fix the Instagram maximum video length. [#39930]
-- Social: Prevent share status styles from affecting other independent elements on the page. [#39991]
-- Subscribe Block: Fix styling for the number of subscribers in the post editor sidebar. [#39929]
+## 14.0 - 2024-11-05
+### Major enhancements
+- Newsletters: Add email previews and a floating subscribe button. [#37722] [#39782]
+- Social: Add support for Bluesky. [#39659] [#39561]
```

Comment:
Are these really "major enhancements"? 🤔

Reply:
I think they're new features that would qualify as major enhancements, we want folks to know about it.